module/Application/view/application/index/index.phtml

```diff
@@ -4,7 +4,7 @@
     <p>
         Congratulations! You have successfully installed the
         <a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">ZF Skeleton Application</a>.
-        You are currently running Zend Framework version <?= \Application\Module::VERSION ?>.
+        You are currently running Zend Framework version <?php echo \Application\Module::VERSION ?>.
```

Comment:
yes, I know, I just want to be consistent - in other views we are always using `<?php echo`